core/tests/app/test_screen.py

```diff
@@ -0,0 +1,27 @@
+from toga_dummy.utils import assert_action_performed
+
+
+def test_name(app):
+    """The name of the screens can be retrieved"""
+    assert app.screens[0].name == "Primary Screen"
+    assert app.screens[1].name == "Secondary Screen"
+
+
+def test_origin(app):
+    """The origin of the screens can be retrieved"""
+    assert app.screens[0].origin == (0, 0)
+    assert app.screens[1].origin == (-1366, -768)
+
+
+def test_size(app):
+    """The size of the screens can be retrieved"""
+    assert app.screens[0].size == (1920, 1080)
+    assert app.screens[1].size == (1366, 768)
+
+
+def test_as_image(app):
```

Comment:
There's no test of creating an image in a different format.

Reply:
Modified the test.